ui/pages/citizen/[tokenId].tsx

```diff
@@ -234,7 +235,7 @@ export default function CitizenDetailPage({
                         )}
                       </div>
                     ) : null}
-                    {address === nft.owner ? (
+                    {/* {address === nft.owner ? (
```

Comment:
Remove commented out code

Reply:
That is for extending the subscription. We should keep that code so that we can easily re-enable it when we're ready.